lib/msal-common/src/error/InteractionRequiredAuthError.ts

```diff
@@ -23,9 +23,19 @@ export const InteractionRequiredAuthSubErrorMessage = [
 ];
 
 /**
- * Error thrown when user interaction is required at the auth server.
+ * Interaction required errors defined by the SDK
  */
-export class InteractionRequiredAuthError extends ServerError {
+export const InteractionRequiredAuthErrorMessage = {
+    noTokensFoundError: {
+        code: "no_tokens_found",
+        desc: "No refresh token found in the cache. Please sign-in."
```

Comment:
Do we want to call out which APIs to use? 
```suggestion
        desc: "No refresh token found in the cache. Please sign-in via ssoSilent, loginRedirect, or loginPopup."
```

Reply:
Might be worth redesigning our errors for v3+ to allow each lib to give more specific error messages. \

> And is it always for refresh tokens only?
Yes. Access tokens missing throws a different error.